torch/onnx/symbolic_opset17.py

```diff
@@ -15,5 +15,19 @@
     SequenceMap
 """
 
+from torch.onnx import symbolic_helper
+
 # EDITING THIS FILE? READ THIS FIRST!
 # see Note [Edit Symbolic Files] in symbolic_helper.py
+
+
+@symbolic_helper.parse_args("v", "is", "v", "v", "f", "i")
+def layer_norm(g, input, normalized_shape, weight, bias, eps, cudnn_enable):
```

Comment:
one more note: Please annotate the type and decorate the function with _beartype.beartype

Reply:
offline meeting: more _beartype.beartype use case will be introduced in another PR from Justin. I have done annotation and beartype on shared layer norm function.